### DISCUSSION TITLE
Nominate Zach Jones to Nebari Contributors Team

### DISCUSSION THREAD

**Discussion by tylergraff** [nebari-dev/nebari#2751] [Community]
I nominate @joneszc to the Contributors team  

Zach has many contributions, RFDs, reviews, and PRs to the Nebari repo:  
https://github.com/nebari-dev/nebari/pull/2723  
https://github.com/nebari-dev/nebari/issues/2604
https://github.com/nebari-dev/nebari/pull/2621
https://github.com/nebari-dev/nebari/pull/2668  
https://github.com/nebari-dev/governance/issues/52
(and others)

**Comment by marcelovilla** [comment]
I support this

**Reply by dcmcand** [reply]
I agree, @dharhas can you add @joneszc ?

**Comment by dharhas** [comment]
done.
